dds_web/api/project.py

```diff
@@ -949,3 +949,28 @@ def put(self):
             "ok": True,
             "message": f"Project {project_id} was set to {'busy' if set_to_busy else 'not busy'}.",
         }
+
+
+class ProjectInfo(flask_restful.Resource):
+    """Get information for a specific project."""
+
+    @auth.login_required
+    @logging_bind_request
+    @handle_validation_errors
+    def get(self):
+        # Get the specified project
+        project = project_schemas.ProjectRequiredSchema().load(flask.request.args)
```

Comment:
Trying to phase this one out so use this instead:
```suggestion
        # Get project ID, project and verify access
        project_id = dds_web.utils.get_required_item(obj=flask.request.args, req="project")
        project = dds_web.utils.collect_project(project_id=project_id)
        dds_web.utils.verify_project_access(project=project)
```